api/strategies/localStrategy.js

```diff
@@ -1,7 +1,17 @@
 const { Strategy: PassportLocalStrategy } = require('passport-local');
 const User = require('../models/User');
 const { loginSchema, errorsToString } = require('./validators');
-const DebugControl = require('../utils/debug.js');
+const winston = require('winston');
+
+// Configure Winston logger
+const logger = winston.createLogger({
+  level: 'info',
+  format: winston.format.combine(winston.format.timestamp(), winston.format.json()),
+  transports: [
+    new winston.transports.Console(),
+    new winston.transports.File({ filename: 'login-logs.log' }),
```

Comment:
here e.g. /logins/login-logs.txt instead of just login-logs.txt
-> for better handling when using logrotate